docs/luigi-element-api.md

```diff
@@ -0,0 +1,157 @@
+<!-- meta
+{
+  "node": {
+    "label": "Luigi Element API",
+    "category": {
+      "label": "API Reference",
+      "collapsible": true
+    },
+    "metaData": {
+      "categoryPosition": 6,
+      "position": 5
+    }
+  }
+}
+meta -->
+
+# Luigi Element
+
+This document outlines the parameters provided by the Luigi Luigi Element. The `LuigiElement` class is a base class for building web component micro frontends within the Luigi framework. It provides essential functionality for rendering and managing the lifecycle of web components.
```

Comment:
Luigi Luigi Element sounds a bit strange ;) 
I'd probably simply remove the whole first sentence

Reply:
I followed the pattern of other API pages - they all start with such sentence 😄 

But ofc I can remove it if you want 😸